astropy/stats/sigma_clipping.py

```diff
@@ -152,7 +152,7 @@ def sigma_clipped_stats(data, mask=None, mask_val=None, sigma=3.0, iters=None):
     Parameters
     ----------
     data : array_like
```

Comment:
Nitpick, but maybe "array-like" is fine?  No need for an underscore since it's not an actual type or anything.